mne/io/eeglab/tests/test_eeglab.py

```diff
@@ -486,11 +486,12 @@ def get_bad_information(eeg, get_pos):
     pos = montage.get_positions()
     n_eeg = 129
     if not has_type:
-        assert pos['nasion'] is None
-        assert pos['lpa'] is None
-        assert pos['rpa'] is None
-        assert len(raw.info['dig']) == n_eeg
-        return
+        # These should now be estimated from the data
+        # TODO: This is in meters... so clearly wrong. (The estimation is
+        # not the problem, the dig are all off like this.)
```

Comment:
a file in testing should be fixed?

Reply:
No idea if the test file is broken or our reading of it. The associated test appears to be written by @agramfort so maybe they'll be able to figure it out most quickly :)